spec/features/pilot/stage_5_check_your_answers_spec.rb

```diff
@@ -22,4 +23,116 @@
     # And can advance through the stages back to 'Confirm your answers'
   end
 
+  def given_i_have_completed_all_the_required_stages
+    complete_destination_stage
+    complete_dates_stage
+    complete_registration_identifier_stage
+    complete_personal_details_stage
+  end
+
+  def and_i_am_on_the_final_check_your_answers_stage
+    visit(stages_check_your_answers_path)
+    expect(page).to have_content("Check your answers")
+  end
+
+  def then_i_see_all_the_questions_together_with_my_answers
+    stages.each do |stage|
+      within ".stage[data-stage='#{stage.name}']" do
+        expect(page).to have_css("h2", text: stage.title)
```

Comment:
yes perhaps it would be clearer to assert: 

`expect(page.find("h2")).to have_text(stage.title)`

i.e. to apply the DOM selector first and then assert the expected content

Reply:
the Capybara "matchers" are listed here: https://www.rubydoc.info/gems/capybara/Capybara/Node/Matchers

and the "finders" here: https://www.rubydoc.info/gems/capybara/Capybara/Node/Finders